web/js/wv.main.js

```diff
@@ -75,6 +75,9 @@ import {debugConfig, debug} from './debug';
 import Brand from './brand';
 import Tour from './tour';
 
+import { default as polyfill } from './polyfill';
+polyfill(); // Polyfills some browser features
```

Comment:
Had to move the polyfills out of `utils/browser` into their own module, which needs to be executed from main.